ps_linklist.php

```diff
@@ -87,7 +88,7 @@ public function __construct()
         $this->description = $this->trans('Adds a block with several links.', array(), 'Modules.Linklist.Admin');
         $this->secure_key = Tools::encrypt($this->name);
 
-        $this->ps_versions_compliancy = array('min' => '1.7.5.0', 'max' => _PS_VERSION_);
+        $this->ps_versions_compliancy = array('min' => '1.7.7.0', 'max' => _PS_VERSION_);
```

Comment:
It's a major BC with we remove  support version.

Reply:
AFAIK, it's not a problem for natives modules.

Reply:
In this case, the version need to be `4.0.0`.
Ping @jolelievre

Reply:
Yep, you're both right

Reply:
<3